airbyte-integrations/connectors/source-sendgrid/acceptance-test-config.yml

```diff
@@ -18,7 +18,7 @@ acceptance_tests:
         status: "failed"
   discovery:
     tests:
-      - config_path: "secrets/old_config.json"
+      - config_path: "secrets/config.json"
```

Comment:
Noting that there seemed to be a very conscious change to switch from `config.json` to `old_config.json` in this PR back in 2022. The only difference in the files I pulled from GCS is `start_time` changing from `"2019-05-20T13:43:57.000Z"` to `"2019-05-20T13:43:57Z"`.

But since it looks like CI tests are passing just fine now, I'm good with the change and thanks for updating. Just adding context in case.

Reply:
Gotcha, thanks for double-checking to verify!